### PR TITLE
Add release notes in docs

### PR DESCRIPTION
**Pull request by stagraqubole**
This is how main page of docs looks like now:
Screenshot 2020-06-11 at 1 33 02 PM

This is how the release notes page looks like now:
Screenshot 2020-06-11 at 1 33 15 PM

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -1,28 +1,20 @@
...
-
-Welcome to RubiX's documentation!
-=================================
+###################
+Rubix Documentation
+###################
 
 RubiX is a light-weight data caching framework that can be used by Big-Data engines.
 RubiX can be extended to support any engine that accesses data in cloud stores using Hadoop FileSystem interface via plugins.
 Using the same plugins, RubiX can also be extended to be used with any cloud store
```
Comment (harmandeeps) [inline docs/index.rst:7]: nit: store -> store.

File: docs/release/release_notes/release-0.3.11.rst [hunk 2]
```diff
@@ -0,0 +1,7 @@
+==============
+Release 0.3.11
+==============
+
+* Fix a regression from 0.3.10 that caused wrong BlockLocations to be returned in CachingFileSystem#listLocatedStatus
+
```
Comment (harmandeeps) [inline docs/release/release_notes/release-0.3.11.rst:6]: nit: Extra line

Comment (harmandeeps) [reply docs/release/release_notes/release-0.3.11.rst:6]: In future, we may also start segregating some improvements based on the engine.

**Comment by harmandeeps** [comment]
> Looks good to me. A few nits.

should we also update copyright from '2019, Qubole' -> '2020, Qubole'?

File: docs/release/release_notes/release-0.3.11.rst (hunk 2 above)
Comment (stagraqubole) [reply docs/release/release_notes/release-0.3.11.rst:6]: Ya, release note page is open form and we can add anything. This change is to just introduce it.

**Comment by stagraqubole** [comment]
> > Looks good to me. A few nits.
> 
> should we also update copyright from '2019, Qubole' -> '2020, Qubole'?

Lets discuss this separately.

**Pull request merged**